README.md

```diff
@@ -170,10 +170,15 @@ takes the following options (proplist):
 * `reconnect_sleep`: integer of milliseconds to sleep between reconnect attempts, default: 100
 * `connect_timeout`: timeout value in milliseconds to use in the connect, default: 5000
 * `socket_options`: proplist of [gen_tcp](https://erlang.org/doc/man/gen_tcp.html)
-  options used when connecting the socket, default is `?SOCKET_OPTS`
+  options used when connecting the socket, default is `?SOCKET_OPTS`.
+  It's possible to specify `{active, N}` or `{active, true}`.
+  Active `once` and `false` are not supported. The default is N = 10.
 * `tls`: enable TLS by providing a list of
   [options](https://erlang.org/doc/man/ssl.html) used when establishing the TLS
-  connection, default is off
+  connection, default is off.
+  It's possible to specify `{active, N}` or `{active, true}` in TLS options.
+  Active `once` and `false` are not supported. The default is N = 10.
+  Note that `{active, N}` with TLS requires OTP 21.3 or later.
```

Comment:
Not important for my project, but just to note: because now the default is `{active, 10}`, and `{active, once}` cannot be provided, this effectively means that to use TLS at all below OTP 21.3, only the `{active, true}` can (and must) be used after this commit, whereas before they could use `{active, once}` (by default). 

Maybe this is worth noting too at least.

Reply:
Changelog seems reasonable to me